mailpoet/lib/Automation/Integrations/Core/Actions/Wait/ValidationResult.php

```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types = 1);
```

Comment:
I like the idea of having each action define its own validation result class because it allows us to work with typed variables. Does this seem like a reasonable approach?

Reply:
I agree overall that this seems unnecessary. I think the main potential benefit is that it could make it easier to write targeted tests, i.e. we would only need to ensure that validation fails a particular way when given a particular problem as opposed to having to set up all the data, try to run the action and then check the results/exceptions.

But as with my other comments, I think this isn't something that needs to be included in this PR, and it would make more sense to introduce additional complexity like this when/if there's a clear need for it.